### ISSUE TITLE
[BUG] cant connect to wallet more than 16 hours. since i installed new linux

### ISSUE THREAD

**Issue by chia888** [Chia-Network/chia-blockchain#4768]
**Describe the bug**
A clear and concise description of what the bug is. (If what you are experiencing is NOT a bug but instead a support issue, please open a Discussion instead!)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. Linux]
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: [e.g. Intel Xeon 8175M]

**Additional context**

**Comment by RinKrul** [comment]
Same issue here.

Linux
Ubuntu 20.04
AMD FX-8370

Using CLI, I am fully synced, plotting and farming, but can not connect to wallet.  If i try chia wallet show all I get is: Connection error. Check if wallet is running at 9256

I have restarted the wallet a few times, the whole node a few times, and the whole computer once.

This started after I had to hard reset the computer because the GUI locked up while it was plotting, farming and fully synced.

**Comment by xie1986** [comment]
i have a same issue

**Comment by ttheekshana** [comment]
`chia start wallet`

**Comment by RinKrul** [comment]
chia start wallet results in: chia_wallet: Already running, use `-r` to restart
                                          chia_full_node: Already running, use `-r` to restart

I have already restarted the wallet several times, as well as the full node.

**Comment by RinKrul** [comment]
Okay, while waiting for my plots to finish, I decided to try something else.

I went into .chia/mainnet/wallet/db and found 6 files.  
I renamed all the files with a .back extension, so they are effectively deleted but also backed up.
Then I restarted the wallet with chia start -r wallet
It said no backup found, and asked if I wanted to skip or use my own backup file.  I chose skip.

And now it appears to be working.  Says I am farming but have no chia, which I expected, but I don't get the connection error anymore.
